### DISCUSSION TITLE
Batch image processing

### DISCUSSION THREAD

**Discussion by sabi3293043** [lllyasviel/ControlNet#65] [Ideas]
i had a png sequance i wanted to process with the same seed but currently can not without doing it 1 by 1 and they are 300 images

**Comment by rasputin2000** [comment]
Agreed, ControlNet is perfect for pose-consistent frame by frame modification, but I cant figure out how to do batching

**Comment by rasamaya** [comment]
apparently they know its not supported. would love to have this too!

**Comment by geroldmeisinger** [comment]
Automatic1111 has batch support in ControlNet -> Batch

**Comment by djbielejeski** [comment]
Not sure if this solves your issue but I've been using this with great success for batching

https://github.com/djbielejeski/folder-batch-automatic1111-extension
